lib/plugsnag/basic_error_report_builder.ex

```diff
@@ -55,6 +55,25 @@ defmodule Plugsnag.BasicErrorReportBuilder do
     |> Keyword.get(field, [])
   end
 
+  defp filter(:query_string, data) when is_binary(data) do
+    {_, data} = data
+    |> String.split("&")
```

Comment:
have you considered using something like https://hexdocs.pm/elixir/URI.html#decode_query/2 to do the decoding for you?

Reply:
Didn't know about this func, it simplifies a lot! Thanks!